pyvista/utilities/geometric_objects.py

```diff
@@ -688,3 +689,68 @@ def CircularArc(pointa, pointb, center, resolution=100, normal=None,
 
     arc.Update()
     return pyvista.wrap(arc.GetOutput())
+
+
+def Pyramid(pointa, pointb, pointc, pointd, pointe):
```

Comment:
You might consider just passing an array or list of points since creating this with 5 args is a bit of a pain.

Reply:
Thanks. It is a good idea.